conf/master

```diff
@@ -1006,6 +1006,10 @@
 # pillar is recompiled and stored. A value of 0 will cause the cache to always be valid.
 #pillar_cache_ttl: 3600
 
+# If and only if a master has set ``pillar_cache: True``, the cache env controls the
+# pillarenv of pillar data, before the cache is considered stored by a master.
+#pillar_cache_env: 'base'
```

Comment:
This should probably be a list, so that you can provide more than just one environment if you wanted to.

Reply:
now dont support cache multi pillarenv, shoule we support ?

Reply:
yes. but seems pillar_cache can only work for one env at the same time.

Reply:
That should not be true, it should be caching the pillar dictionary on a per minion basis.

Looking at this closer, i am not sure this is a good approach to this, because any case that this is tried to be used without a pillarenv set, and just environments assigned in the top file, this will not work, which is bad because using environments from a top file is the default.

Reply:
hmm, i think i found a bug. i will commit another pr for fix it.